test/main.sh

```diff
@@ -192,6 +192,8 @@ run_test() {
 	${TEST_CURRENT}
 	END_TIME="$(date +%s)"
 
+	collect_go_cover_files
```

Comment:
Why is it not sufficient to do the collection here but also in `reset_systems` and `restore_systems`?

Reply:
A few of the test suites use `reset_systems` during test cases. `reset_systems` may call `restore_systems` when `SNAPSHOT_RESTORE` is set to `1`.